.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,92 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        #  ** Update flake8 literals if change os or python-versions
+        os: [ubuntu-latest, windows-latest] # , macos-latest]
+        python-version:  [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I'm a bit curious about this. Why v1?

Reply:
Just accidental - probably from whatever code I 'borrowed' for that part.  I'll change to v2.